mteb/tasks/Retrieval/jpn/JaQuADRetrieval.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from datasets import load_dataset
+
+from mteb.abstasks import AbsTaskRetrieval, TaskMetadata
+
+
+class JaQuADRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="JaQuADRetrieval",
+        dataset={
+            "path": "SkelterLabsInc/JaQuAD",
+            "revision": "05600ff310a0970823e70f82f428893b85c71ffe",
+        },
+        description="Human-annotated question-answer pairs for Japanese wikipedia pages.",
+        reference="https://arxiv.org/abs/2202.01764",
+        type="Retrieval",
+        category="p2p",
+        eval_splits=["validation"],
+        eval_langs=["jpn-Jpan"],
+        main_score="ncdg_at_10",
+        date=("2022-01-01", "2022-12-31"),  # approximate guess
+        form=["written"],
+        domains=["Encyclopaedic", "Non-fiction"],
+        task_subtypes=["Question answering"],
+        license="CC-BY-SA-3.0",
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
+        dialect=None,
+        text_creation="found",
+        bibtex_citation="""@misc{so2022jaquad,
+    title={{JaQuAD: Japanese Question Answering Dataset for Machine Reading Comprehension}},
+    author={ByungHoon So and Kyuhong Byun and Kyungwon Kang and Seongjin Cho},
+    year={2022},
+    eprint={2202.01764},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}""",
+        n_samples={"validation": 3939},
```

Comment:
Any reason why this dataset is not downsampled?

Reply:
Fixed in #352 and now synced to it. Thanks!

Reply:
will update and resync this once #352 is resolved.